gnocchi/json.py

```diff
@@ -30,8 +30,7 @@ def to_primitive(obj):
     if isinstance(obj, datetime.datetime):
         return obj.isoformat()
     if isinstance(obj, numpy.datetime64):
-        # Do not include nanoseconds if null
-        return str(obj).rpartition(".000000000")[0] + "+00:00"
+        return numpy.datetime_as_string(obj, unit='s') + "+00:00"
```

Comment:
```suggestion
        return str(obj.astype('datetime64[s]')) + "+00:00"
```
is a bit faster but this serializer doesn't seem particularly efficient anyways so it doesn't matter.

Reply:
I see. I do find the way it is right now easier to read, and more direct to understand. However, if you guys prefer this other way, I can change it.

Reply:
lgtm too. datetime_as_string seems to be part of stable api from what i can tell.